tools/deprecate_version/deprecate_version.py

```diff
@@ -90,47 +90,61 @@ def CreateIssues(access_token, runtime_and_pr):
     raise DeprecateVersionError('Unknown labels (expected %s, got %s)' % (LABELS, labels))
 
   issues = []
-  for runtime_guard, pr in runtime_and_pr:
+  for runtime_guard, pr, commit in runtime_and_pr:
     # Who is the author?
-    pr_info = repo.get_pull(pr)
+    if pr:
+      # Extract PR title, number, and author.
+      pr_info = repo.get_pull(pr)
+      change_title = pr_info.title
+      number = ('#%d') % pr
+      login = pr_info.user.login
+    else:
+      # Extract Commit message, sha, and author.
+      # In this case we cannot add a user login for an assignee.
+      change_title = commit.message.split('\n')[0]  # Remove sign-off mesage
```

Comment:
It's not sign off but description too (if there is one) yes?

Reply:
Yep -- that's right it.